_blueTheme.scss

```diff
@@ -443,7 +445,8 @@
         .pxb-drawer-nav-item-active .pxb-info-list-item .pxb-info-list-item-icon-wrapper {
             color: map-get($primary, 500);
         }
-        .pxb-info-list-item .mat-list-item-content .mat-list-icon {
+
+        .pxb-info-list-item .pxb-info-list-item-icon-wrapper  {
```

Comment:
I've lowered the specificity here so it retains black[500] on inactive, but doesn't override the active styles.